### PR TITLE
added categories page

### PR DESCRIPTION
**Pull request by Shah-Arsalan**

### CODE REVIEW

**Review by samyakshah3008 — APPROVED** [review]
Looks good to merge!

File: WebPages/category/category.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  object-fit: fill;
+  width: 100%;
+  border-radius: 1rem;
+}
+
+.card-skeleton {
+  border-radius: 0.5rem;
+  height: 17rem;
+  border-radius: 1rem;
+  background-color: #272626;
```
Comment (Rohanmond) [inline WebPages/category/category.css]: use color var

File: WebPages/category/category.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            </div>
+
+
+
+
+            <div class="card-skeleton ">
+
+                <div class="imageContainer position-relative">
+                    <img class="image"
+                        src="https://img.jagranjosh.com/imported/images/E/GK/ICC-Cricket-World-Cup-2023.png" alt="">
```
Comment (Rohanmond) [inline WebPages/category/category.html]: give alt

**Review by Rohanmond — APPROVED** [review]

File: WebPages/category/category.css (hunk 1 above)
Comment (Shah-Arsalan) [reply WebPages/category/category.css]: okay changing

File: WebPages/category/category.html (hunk 2 above)
Comment (Shah-Arsalan) [reply WebPages/category/category.html]: done

**Pull request merged**